Editor/Tabs/LayersTab.cs

```diff
@@ -277,39 +289,55 @@ public void SetDefault(bool value)
             }
         }
 
-        public void ToggleParameterSync(SyncedParameter parameter, bool toggle)
```

Comment:
To avoid to pull a VRC we should not remove/change public method definitions without some deprecation time
So readd the previous definition with an [Obsolete] attribute and from that one you call the new one with the "isExpression" bool be the same value of the "synced" one passed down via the "toggle" param of the old method, so we should be keeping the same effective functionality until we will completely deprecate the method call a couple of releases down the road